openshift/system/Dockerfile

```diff
@@ -7,17 +7,19 @@ ENV RUBY_MAJOR_VERSION=2 \
 ENV RUBY_VERSION="${RUBY_MAJOR_VERSION}.${RUBY_MINOR_VERSION}"
 ADD . ./
 ADD openshift/system/config/* ./config/
-RUN dnf -y module enable ruby:${RUBY_VERSION} nodejs:16 mysql:8.0 \
-    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info make automake gcc gcc-c++ redhat-rpm-config postgresql rubygem-irb rubygem-rdoc ruby-devel nodejs libpq-devel mysql-devel gd-devel git 'dnf-command(download)' podman-catatonit \
-    && gem install bundler --version 2.2.25 \
+RUN dnf -y --setopt=module_stream_switch=True module enable ruby:${RUBY_VERSION} nodejs:16 mysql:8.0 \
+    && dnf install -y --setopt=skip_missing_names_on_install=False,tsflags=nodocs shared-mime-info make automake gcc gcc-c++ redhat-rpm-config postgresql rubygem-irb rubygem-rdoc ruby-devel nodejs libpq-devel mysql-devel gd-devel libxml2-devel libxslt-devel git 'dnf-command(download)' podman-catatonit \
+    && BUNDLER_VERSION=$(awk '/BUNDLED WITH/ { getline; print $1 }' Gemfile.lock) \
+    && gem install --no-document bundler:$BUNDLER_VERSION \
+    && bundle config build.nokogiri --use-system-libraries \
```

Comment:
in production we build with the RHEL libxml2 and xslt libs. So do this here as well